example/src/App.tsx

```diff
@@ -264,6 +283,15 @@ const AppContainer = () => {
                 style={styles.simformImage}
                 resizeMode="contain"
               />
+              <Pressable
+                style={styles.playBackControlPressable}
```

Comment:
Let's see if merging https://github.com/SimformSolutionsPvtLtd/react-native-audio-waveform/pull/118 first fix that issue!

Reply:
Hey, since blocking PR got merged, I continue.
Finally, I move that in the scroll list, and even better, I hide that under a click on the header.
So now, to show "advanced options," you need to click on the SIMFORM header and see the information below.

With this, we will be able to hide those options on first experience (including the delete all recordings from [this PR](https://github.com/SimformSolutionsPvtLtd/react-native-audio-waveform/pull/122) once merged), but still have them to test out!

<img width="307" alt="image" src="https://github.com/user-attachments/assets/ea768322-2fb1-4cbd-a12d-97f172591414">

Reply:
I merge the `Delete recorded audio files` and also hide it under this "advanced option" hidden under a click on the simform header

Reply:
FYI: I found the icon `tintColor` issue; it was the colour space set to Gray. Once changed to RGB, it now iOS render the `tintColor` correctly